tests/CMakeLists.txt

```diff
@@ -1,21 +1,12 @@
-cmake_minimum_required(VERSION 2.8)
-project(mustache)
+add_executable(mustache-unit-tests
+	tests.cpp
+)
+
+target_link_libraries(mustache-unit-tests PRIVATE mustache)
+
 if (UNIX)
-	add_definitions(
-		-Wall
-		-Wextra
-		-Werror
-		-std=c++11
-	)
+    target_compile_options(mustache-unit-tests PRIVATE -Wall -Wextra -Werror)
 elseif (MSVC)
-	add_definitions(
-		/W3
-		/WX
-	)
+    target_compile_options(mustache-unit-tests PRIVATE /W3 /WX)
 endif()
-add_executable(mustache
-	../mustache.hpp # to show in IDE
```

Comment:
oops, I realized this is useful to have these sources show up in the IDE, such as Xcode. Can you add them back in a new PR?

Reply:
Yes I will do that